packages/components/src/popover/index.js

```diff
@@ -194,6 +194,7 @@ const Popover = (
 		shift( {
 			crossAxis: true,
 			limiter: limitShift(),
+			padding: 1, // Necessary to avoid flickering at the edge of the viewport.
```

Comment:
I'm not sure what it is about the particular settings we're using, but I couldn't get everything to work correctly (removed maxWidth on the size middleware, and a popover at the edge of the screen) without adding at least `1px` of padding to the `shift` middleware. Otherwise We'd see jiggling like in this comment: https://github.com/WordPress/gutenberg/pull/41268#discussion_r881293934

<img src="https://user-images.githubusercontent.com/14988353/170201659-b54846e2-b059-4541-b4e7-48878c0e71c0.gif" />

I'm hoping that adding `1px` padding here (which only gets applied at the edge of the viewport) will be viable in the shorter-term. Though it'd be good to get to the bottom of why the issue occurs without the padding.